engine/src/flutter/shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -393,7 +394,8 @@ void Engine::Initialize(
             auto platform_view = shell_->GetPlatformView();
 
             if (platform_view) {
-              platform_view->DispatchSemanticsAction(node_id, action, {});
+              platform_view->DispatchSemanticsAction(kImplicitViewId, node_id,
```

Comment:
Let's add a TODO here:

```suggestion
              // TODO(fuchsia): Remove implicit view assumption.
              // https://github.com/flutter/flutter/issues/142845
              platform_view->DispatchSemanticsAction(kImplicitViewId, node_id,
```